source/winAPI/sessionTracking.py

```diff
@@ -106,110 +90,105 @@ class WindowsTrackedSession(enum.IntEnum):
 	SESSION_TERMINATE = 11
 
 
-_toggleWindowsSessionStatePair: Dict[WindowsTrackedSession, WindowsTrackedSession] = {
-	WindowsTrackedSession.CONSOLE_CONNECT: WindowsTrackedSession.CONSOLE_DISCONNECT,
-	WindowsTrackedSession.CONSOLE_DISCONNECT: WindowsTrackedSession.CONSOLE_CONNECT,
-	WindowsTrackedSession.REMOTE_CONNECT: WindowsTrackedSession.REMOTE_DISCONNECT,
-	WindowsTrackedSession.REMOTE_DISCONNECT: WindowsTrackedSession.REMOTE_CONNECT,
-	WindowsTrackedSession.SESSION_LOGON: WindowsTrackedSession.SESSION_LOGOFF,
-	WindowsTrackedSession.SESSION_LOGOFF: WindowsTrackedSession.SESSION_LOGON,
-	WindowsTrackedSession.SESSION_LOCK: WindowsTrackedSession.SESSION_UNLOCK,
-	WindowsTrackedSession.SESSION_UNLOCK: WindowsTrackedSession.SESSION_LOCK,
-	WindowsTrackedSession.SESSION_CREATE: WindowsTrackedSession.SESSION_TERMINATE,
-	WindowsTrackedSession.SESSION_TERMINATE: WindowsTrackedSession.SESSION_CREATE,
-}
-"""
-Pair of WindowsTrackedSession, where each key has a value of the opposite state.
-e.g. SESSION_LOCK/SESSION_UNLOCK.
-"""
+class _CurrentSessionState(AutoPropertyObject):
+
+	def __init__(self) -> None:
+		self._windowsLockStateIsUnknown = False
+		"""
+		Track if any 'Unknown' Value when querying the Session Lock status has been encountered.
+		Resets when a successful Windows lock state query completes.
+		"""
+		self._prev_isWindowsLockedValue = False
+		"""Before NVDA initialization, assume Windows is unlocked.
+		This means if Windows is locked, NVDA processes the state change correctly.
+		"""
+		super().__init__()
+
+	_cache_isWindowsLocked = True
+
+	# typing information for auto property _get_isWindowsLocked
+	isWindowsLocked: bool
+
+	def _get_isWindowsLocked(self) -> bool:

```

Comment:
Given this is new, can we change to the name to `isOnLockScreen` to avoid some confusion about sign-on screen?